apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -65,16 +71,42 @@ const CurriculumCatalog = ({
     setAssignSuccessMessage('');
   };
 
-  const handleExpandedCardChange = key => {
-    if (expandedCardKey !== key) {
+  const handleQuickViewClicked = key => {
+    if (expandedCardKey === key) {
```

Comment:
To make sure I'm understanding... we know a user is closing an expanded card if the 'key' of the click event matches the current expanded card?

Reply:
Yep!